client/sections.js

```diff
@@ -291,7 +298,7 @@ const sections = [
 	// this MUST be the first section for /read paths so subsequent sections under /read can override settings
 	{
 		name: 'reader',
-		paths: [ '/', '/read' ],
```

Comment:
You might need to update some tests that are failing because of that.

Reply:
I fixed the tests